src/OrchardCore.Modules/OrchardCore.OpenIdConnect/Services/IOpenIdConnectService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Mvc.ModelBinding;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OrchardCore.OpenIdConnect.Services
+{
+    public interface IOpenIdConnectService
```

Comment:
This should be in an `Abstractions` class library as it is used by other modules.